Docker-Swarm-deployment/single-node/catalogue/example-secrets/secrets/cat-summary-config.json

```
@@ -0,0 +1,10 @@
+{
+    "username": "iudx_rs_user",
+    "password": "<cat iudx-deployment/K8s-deployment/Charts/postgresql/secrets/passwords/postgres-rs-password>",
+    "host": "localhost",
```

Since it is running in swarm, host will always be tasks.postgres right?
```suggestion
    "host": "tasks.postgres",
```